test/test_helper.rb

```diff
@@ -20,4 +21,37 @@ class ActionController::Base
 class ActionDispatch::IntegrationTest
   # Make the Capybara DSL available in all integration tests
   include Capybara::DSL
+  include Slimmer::TestHelpers::SharedTemplates
+
+  def assert_has_component_metadata_pair(label, value)
+    within shared_component_selector("metadata") do
+      # Flatten top level / "other" args, for consistent hash access
+      component_args = JSON.parse(page.text).tap do |args|
+        args.merge!(args.delete("other"))
+      end
+      assert_equal value, component_args.fetch(label)
+    end
+  end
+
+  def assert_has_component_title(title)
+    within shared_component_selector("title") do
+      assert_equal title, JSON.parse(page.text).fetch("title")
+    end
+  end
+
+  def setup_and_visit_content_item(name)
+    @content_item = JSON.parse(get_content_example(name)).tap do |item|
+      content_store_has_item(item["base_path"], item.to_json)
+      visit item["base_path"]
+    end
+  end
+
+  def get_content_example(name)
+    GovukContentSchemaTestHelpers::Examples.new.get(schema_format, name)
+  end
+
+  def schema_format
+    # Override this method if your test file doesn't match the convention
```

Comment:
This may be super-niggly but perhaps this comment should go above the method signature since that is what it applies to?